### PR TITLE
docs(example, android): add documentation on native development for a…

### PR DESCRIPTION
**Pull request by jspizziri**
…ndroid

### CODE REVIEW

File: example/README.md [hunk 1]
```diff
@@ -63,4 +63,71 @@ screenshots below).
...
+
+3. Point your RNTP dependency at the local build:
+
+```groovy
+// react-native-track-player/android/build.gradle
+...
+
+dependencies {
+//    implementation 'com.github.doublesymmetry:kotlin-audio:0.1.33' // this is remote
+    implementation 'com.github.DoubleSymmetry:KotlinAudio:v0.1.33' // this is local
```
Comment (mattleff) [inline example/README.md:117]:
```suggestion
//    implementation 'com.github.doublesymmetry:kotlin-audio:0.1.33' // this is local
    implementation 'com.github.DoubleSymmetry:KotlinAudio:v0.1.33' // this is remote
```

@jspizziri The comments seem backward to me. I had to use the hyphenated name to get the local build. 

Also, I had to add `mavenLocal()` to `example/android/build.gradle` (like `allprojects { repositories { mavenLocal() ... } }`). I might be doing something wrong, but I needed those changes (and the fix in #1552) to be able to test with the local build of KotlinAudio.

Comment (jspizziri) [reply example/README.md:117]: @mattleff thanks for that. I'm going to take a look into that and I'll update the docs as necessary!

Comment (jspizziri) [reply example/README.md]: @mattleff I've just updated the docs based on your feedback. If you have time I'd appreciate if you took another look. Seemed to work for me.

File: example/README.md [hunk 2]
```diff
@@ -61,6 +58,94 @@ screenshots below).
...
+        maven {
+...
+```
+
+
+#### 5. Install the new version of RNTP in the example app and build android:
+
+```sh
+cd ./example
+yarn add file:..
```
Comment (mattleff) [inline example/README.md:147]:
```suggestion
yarn add react-native-track-player@file:..
```

**Review by mattleff — APPROVED** [review]
LGTM! This matches what I had to do. Thanks for writing this up!

File: example/README.md (hunk 2 above)
Comment (jspizziri) [reply example/README.md:147]: @mattleff hmmm.. `yarn add file:..` seems to work for me, is that not a thing?

Comment (mattleff) [reply example/README.md:147]: @jspizziri Nvm, it works fine. 😜

**Review by mpivchev — APPROVED** [review]
Awesome :)

**Pull request merged**